### DISCUSSION TITLE
Does each magick file in temporary path denoting each failure file ?

### DISCUSSION THREAD

**Discussion by akanksha7054** [ImageMagick/ImageMagick#6331] [Help] [answered]
### ImageMagick version

7_1_0_60

### Operating system, version and so on

linux centOS 7

### Description

 this is policy.xml configuration
   policy domain="resource" name="memory" value="1GiB"
   policy domain="resource" name="map" value="2GiB"
   policy domain="resource" name="time" value="300"
   policy domain="resource" name="temporary-path" value="/user/local/magick_temp"
   policy domain="resource" name="disk" value="5GiB"

we have almost 76K temp magick file in temporary path folder. i want to know what it denotes.
1- is 76K files got failed. 
2- for each file we are generating only one magick temp file ?

**Comment by snibgo** [comment, marked as answer]
You don't need to keep the temporary files. You might, for example, delete them every day.

If IM need to create a pixel cache and there is not enough memory, it will use disk (a temmporary file). One command may need more than one pixel cache. When the command finishes, it will delete the temporary files. This will happen even if the command has an error. If the command is killed externally, it has no chance to delete the files.

Your disk limit is 5 GiB, not 5 MiB.1 GiB = 1024 MiB.
